jobs-builder/jobs/pr.yaml

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains the configurations to generate the pull request and
+# baseline jobs.
+#
+---
+###
+# Define shareable YAML snippets.
+###
+- main_jobs_common_properties: &main_jobs_common_properties
+    name: 'common_job_properties'
+    project-type: freestyle
+    disabled: false
+    concurrent: {concurrent_toggle}
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 30
+    # Convert the os variable to label name.
+    node: !include-jinja2: include/os2node.yaml.inc
+    wrappers:
+      - ansicolor:
+          colormap: "xterm"
+      - openstack:
+          single-use: True
+      - timestamps
+      - timeout:
+          timeout: 20
+          type: no-activity
+- main_jobs_common_publishers: &main_jobs_common_publishers
+    name: 'default_publishers'
+    publishers:
+      - post-tasks:
+          - matches:
+              - log-text: .*
+                operator: OR
+            script: |
+              #!/bin/bash
```

Comment:
Again, why not add the standard shell header here too?

Random thought: I wonder if we need a `.ci/kata_header.sh` to make this easy to source and minimise duplication.

Reply:
Here is yet another thin wrapper, this time to https://raw.githubusercontent.com/kata-containers/tests/main/.ci/teardown.sh . I just realized that unlike `ci_entry_point.sh`, the `teardown.sh` script doesn't set those shell properties. Maybe it is by design because we don't want it to fail early? (it collects logs; one or more commands might fail and it is ok?)

Reply:
Regarding adding the `.ci/kata_header.sh`; then people like me will forget to source it :)